concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -13,6 +13,7 @@ groups:
       - smoke-tests
```

Comment:
nothing wrong with this line, there's a typo in commit message 'out' instead of 'our'
github should allow commenting on PR messages as well